tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/browsers/servo.py

```diff
@@ -54,7 +54,7 @@ def env_extras(**kwargs):
 def env_options():
     return {"host": "127.0.0.1",
             "external_host": "web-platform.test",
-            "bind_hostname": "false",
+            "bind_hostname": "true",
```

Comment:
This change got reverted. It probably should not have been folded into the previous big wptup commit.